components/library/libpeas/Makefile

```diff
@@ -56,6 +57,7 @@ CONFIGURE_ENV.64 += PYTHON3_CONFIG=/usr/bin/$(MACH64)/python3.4-config
 CONFIGURE_OPTIONS += --disable-static
 CONFIGURE_OPTIONS += --enable-gtk
 CONFIGURE_OPTIONS += --enable-lua5.2
+CONFIGURE_OPTIONS += --disable-luajit
```

Comment:
This is not strictly required, luajit is not found and so disabled in libpeas (as I see from configure)

Reply:
Ah, OK, I just don't have it installed.